source/simulator/helper_functions.cc

```diff
@@ -1725,6 +1725,93 @@ namespace aspect
   }
 
 
+  template <int dim>
+  void Simulator<dim>::interpolate_material_output_into_field ()
+  {
+    // we need some temporary vectors to store our updates to composition in
+    // before we copy them over to the solution vector in the end
+    LinearAlgebra::BlockVector distributed_vector (introspection.index_sets.system_partitioning,
+                                                   mpi_communicator);
+
+    pcout << "   Copying material properties into prescribed compositional fields."
```

Comment:
I find the `material properties` a bit confusing, this sounds to the user as if you would copy things like density or viscosity. What about `Copying properties into prescribed compositional fields`? Then the user would need to know themselves what is copied.